src/core_plugins/kibana/public/management/sections/objects/_objects.js

```diff
@@ -178,71 +193,85 @@ uiModules.get('apps/management')
           return;
         }
 
-        return new Promise((resolve) => {
-          confirmModal(
```

Comment:
It looks like the functionality to choose if you want to be prompted to automatically overwrite, or be prompted each time was removed.

Reply:
Hm. I thought we wanted that behavior removed? Is that not the case anymore?